.travis.yml

```diff
@@ -1,188 +1,109 @@
-        - .travis/launch-http-server || travis_terminate 1
-        - ./gradlew --parallel test integTest jacocoTestReport || travis_terminate 1
-        - bash <(curl -s https://codecov.io/bash)  # upload coverage report - https://github.com/codecov/example-gradle
-    - stage: verify
-      env: "Smoke Test"
-      jdk: openjdk11
-      cache:
-        directories:
-          - '$HOME/.gradle'
-          - '.gradle'
-          - '~/jdk-11.0.5+10'
+    
+    ########################################################################
+    # Jobs are ordered tests first and then slower jobs before faster ones.
+  
+    - stage: Verify
+      name: "Tests: Integration Test"
+      addons: { postgresql: "10" }
+      script: [ $VERIFY_STAGE/integration-testing/run || $REPORT_FAIL ]
+
+    - stage: Verify
+      name: "Tests: Unit Test"
+      script: [ $VERIFY_STAGE/unit-testing/run || $REPORT_FAIL ]
+    
+    - stage: Verify
+      name: "Tests: Smoke Test"
       addons: {postgresql: "10"}
-      before_install:
-        - ./.travis/install_java_11.0.5
-        - export JAVA_HOME=~/jdk-11.0.5+10
-        - export PATH=${JAVA_HOME}/bin:$PATH
-        - java --version
-      install: skip
-      script:
-        - ./.travis/setup_database
-        - ./.travis/run_smoke_tests
-    - stage: deploy artifacts
-      env: "Upload to Github Releases"
+      script: [ $VERIFY_STAGE/smoke-testing/run || $REPORT_FAIL ]
+
+    - stage: Verify
+      name: "Analysis: PMD Verification"
+      script: [ $VERIFY_STAGE/pmd-verification/run || $REPORT_FAIL ]
+    
+    - stage: Verify
+      name: "Analysis: Checkstyle"
+      script: [ $VERIFY_STAGE/checkstyle/run || $REPORT_FAIL ]
+    
+    - stage: Verify
+      name: "Analysis: Java Formatting"
+      script: [ $VERIFY_STAGE/java-formatting/run || $REPORT_FAIL ]
+    
+    - stage: Verify
+      name: "Analysis: Check Links and YAML Syntax"
+      script: [ $VERIFY_STAGE/check-links-and-yaml-syntax/run || $REPORT_FAIL ]
+      
+    - stage: Verify
+      name: "Analysis: Code Convention Checks"
+      script: [ $VERIFY_STAGE/code-convention-checks/run || $REPORT_FAIL ]
+
+    - stage: Verify
+      name: "Analysis: Find Unused or Unmatched Tests"
+      script: [ $VERIFY_STAGE/find-unused-or-unmatched-tests/run || $REPORT_FAIL ]
+
+    ########################################################################
+
+    - stage: Deploy
+      name: "Push Map Descriptions to Website"
+      if: (branch = master) and (repo = 'triplea-game/triplea') and (type != 'pull_request')
+      addons: { apt: { packages: [ python3, python3-yaml ] } }
+      script: [ $DEPLOY_STAGE/update-maps-on-website/run || $REPORT_FAIL ]     
+
+    - stage: Deploy
+      name: "Ansible Deploy to Prerelease"
+      if: (branch = master) and (repo = 'triplea-game/triplea') and (type != 'pull_request')
+      language: python
+      python: "3.6"
+      addons: { apt: { packages: [ sshpass ] } }
+      install: [ pip install ansible ]
```

Comment:
The same thing here, similar to the script thingy. It shouldn't be necessary to add list brackets here.